Bluejay.asm

```diff
@@ -3959,26 +4096,16 @@ motor_start:
 
 	call	wait1ms
 
-	; Read initial average temperature
-	Start_Adc						; Start adc conversion
-
-	jnb	ADC0CN0_ADINT, $			; Wait for adc conversion to complete
-
-	mov	Current_Average_Temp, ADC0L	; Read initial temperature
-	mov	A, ADC0H
-	jnz	($+5)					; Is reading below 256?
-	mov	Current_Average_Temp, #0		; Yes - set average temperature value to zero
-
-	mov	Adc_Conversion_Cnt, #8		; Make sure a temp reading is done
-	call	check_temp_and_limit_power
-	mov	Adc_Conversion_Cnt, #8		; Make sure a temp reading is done next time
+	mov Ext_Telemetry_H, #0				; Clear extended telemetry data
+	mov	Temp_Pwm_Level_Setpoint, #255	; Initialize temperature pwm level setpoint
```

Comment:
This mov is useless because we are writing Pwm_Limit_Beg into Temp_Pwm_Level_Setpoint right below this.

Reply:
Fixed